### PR TITLE
feat: added grids

### PR DESCRIPTION
**Pull request by aryan-13**
- added grids with 50-50 ratio
- added grids with 70-30 ratio
- added grids with 30-70 ratio

### CODE REVIEW

File: components/grids.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+.grid-30-70{
+    display: grid;
+    grid-template-columns: 30% 70%;
+}
+.grid-70-30{
+    display: grid;
+    grid-template-columns: 70% 30%;
+}
+.grid-item-1{
+    background-color: #F7F7DA;
```
Comment (anshikaa12) [inline components/grids.css:14]: use variables for colors!

**Review by anshikaa12 — APPROVED** [review]
good to merge

File: components/grids.css (hunk 1 above)
Comment (aryan-13) [reply components/grids.css:14]: I won't reuse these variables, therefore I used the actual code

**Review by Shivanshsharma01 — APPROVED** [review]
The code is clean with good use of variables. Can merge

**Pull request merged**